src/main/app/src/Components/LandscapeComponent/LandscapeDashboard/LandscapeDashboard.tsx

```diff
@@ -17,10 +17,38 @@ const LandscapeDashboard: React.FC = () => {
   const [sliderContent, setSliderContent] = useState<string | ReactElement | null>(null);
   const [showSlider, setShowSlider] = useState(false);
   const [assessments, setAssessments] = useState<IAssessment | null>(null);
+  const [highlightElement, setHighlightElement] = useState<Element | HTMLCollection | null>(null);
+
+  const findItem = (fullyQualifiedItemIdentifier: string) => {
+    const element = document.getElementById(fullyQualifiedItemIdentifier);
+    setHighlightElement(element);
+  };
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
+
+    if (highlightElement instanceof Element) {
+      highlightElement.classList.add('highlightDot');
+      highlightElement.scrollIntoView({ behavior: 'smooth', block: 'end' });
+      const itemGroup = highlightElement.parentElement?.parentElement?.parentElement;
```

Comment:
das muss ich nach dem wochenende noch überarbeiten :D

Reply:
What language is this? 😄